cli/src/semgrep/rule_lang.py

```diff
@@ -463,6 +463,7 @@ class RuleValidation:
         "pattern-either",
         "pattern-regex",
         "patterns",
+        "semgrep-internal-patterns-allow-disjoint",
```

Comment:
Does this need updated to be "anywhere"?

Reply:
Think this can just be removed with the update since this shouldn't be permitted to appear at top level.